src/core/Akka.Cluster.Tests.MultiNode/SurviveNetworkInstabilitySpec.cs

```diff
@@ -0,0 +1,555 @@
+    public class SurviveNetworkInstabilityMultiNode2 : SurviveNetworkInstabilitySpec { }
+    public class SurviveNetworkInstabilityMultiNode3 : SurviveNetworkInstabilitySpec { }
+    public class SurviveNetworkInstabilityMultiNode4 : SurviveNetworkInstabilitySpec { }
+    public class SurviveNetworkInstabilityMultiNode5 : SurviveNetworkInstabilitySpec { }
+    public class SurviveNetworkInstabilityMultiNode6 : SurviveNetworkInstabilitySpec { }
+    public class SurviveNetworkInstabilityMultiNode7 : SurviveNetworkInstabilitySpec { }
+    public class SurviveNetworkInstabilityMultiNode8 : SurviveNetworkInstabilitySpec { }
+
+    public abstract class SurviveNetworkInstabilitySpec : MultiNodeClusterSpec
+    {
+        private readonly SurviveNetworkInstabilitySpecConfig _config;
+
+        protected SurviveNetworkInstabilitySpec()
+            : this(new SurviveNetworkInstabilitySpecConfig())
+        {
+        }
+
+        protected SurviveNetworkInstabilitySpec(SurviveNetworkInstabilitySpecConfig config) : base(config)
+        {
+            _config = config;
+            Sys.ActorOf<SurviveNetworkInstabilitySpecConfig.Echo>("echo");
+        }
+
+        private void AssertUnreachable(params RoleName[] subjects)
+        {
+            var expected = subjects.Select(c => GetAddress(c)).ToImmutableHashSet();
+            AwaitAssert(() => ClusterView.UnreachableMembers
+                .Select(c => c.Address).Should().BeEquivalentTo(expected));
+        }
+
+        private void AssertCanTalk(params RoleName[] alive)
+        {
+            RunOn(() =>
+            {
+                AwaitAllReachable();
+            }, alive);
+            EnterBarrier("reachable-ok");
+
+            RunOn(() =>
+            {
+                foreach (var to in alive)
+                {
+                    var sel = Sys.ActorSelection(Node(to) / "user" / "echo");
+                    var msg = $"ping-{to}";
+                    var p = CreateTestProbe();
+                    AwaitAssert(() =>
+                    {
+                        sel.Tell(msg, p.Ref);
+                        p.ExpectMsg(msg, 1.Seconds());
+                    });
+                    p.Ref.Tell(PoisonPill.Instance);
+                }
+            }, alive);
+            EnterBarrier("ping-ok");
+        }
+
+        [MultiNodeFact]
+        public void SurviveNetworkInstabilitySpecs()
+        {
+            A_Network_partition_tolerant_cluster_must_reach_initial_convergence();
+            A_Network_partition_tolerant_cluster_must_heal_after_a_broken_pair();
+            A_Network_partition_tolerant_cluster_must_heal_after_one_isolated_node();
+            A_Network_partition_tolerant_cluster_must_heal_two_isolated_islands();
+            //A_Network_partition_tolerant_cluster_must_heal_after_unreachable_when_ring_is_changed();
+            A_Network_partition_tolerant_cluster_must_down_and_remove_quarantined_node();
```

Comment:
I could run only one of these two spec simultaneously. Not both. It is weird, because half a year ago these two specs worked fine